### PR TITLE
Add currency calculation to LOS records endpoint

### PR DESCRIPTION
**Pull request by Phazz**

### CODE REVIEW

File: content/reference/endpoints/los_records.md [hunk 1]
```diff
@@ -12,25 +12,25 @@
...
-max_occupancy        | Integer | Read       | Maximum occupancy for which the LOS record is valid.
-original_currency    | String  | Read       | Currency in which the rates are originating from.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
-currency             | String  | Read       | Currency in which the rates are in the response.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
-exchange_rate        | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`.
+day                  | [Date](/reference/enums#formats) | Read       | Day for which this LOS record is valid.
+min_occupancy        | Integer | Read       | Minimum occupancy for which this LOS record is valid.
+max_occupancy        | Integer | Read       | Maximum occupancy for which this LOS record is valid.
+original_currency    | String  | Read       | Currency of the rate being sent to BookingSync.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
```
Comment (Phazz) [inline content/reference/endpoints/los_records.md]: Trying to clarify this sentence. I do not know the context so might need reformulation.

**Comment by Phazz** [comment]
@Azdaroth Please review and merge

File: content/reference/endpoints/los_records.md (hunk 1 above)
Comment (Azdaroth) [inline content/reference/endpoints/los_records.md]: Yeah, that should be like "original currency that LOS record was generated with (useful when using `exchange_to_currency`query param)"

File: content/reference/endpoints/los_records.md [hunk 2]
```diff
@@ -12,25 +12,25 @@
...
+currency             | String  | Read       | Currency of the rate in the response.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
```
Comment (Azdaroth) [inline content/reference/endpoints/los_records.md]: Would be better as "requested currency of the LOS records, by default same as `original_currency`"

File: content/reference/endpoints/los_records.md [hunk 3]
```diff
@@ -12,25 +12,25 @@
...
+exchange_rate        | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from `original_currency` to `currency` by calculating `currency = original_currency * exchange_rate`.
```
Comment (Azdaroth) [inline content/reference/endpoints/los_records.md]: I don't think we need to include anything about the way it is calcualted, I think it's pretty obvious what exchange rate is when you are actually using currency exchange.

File: content/reference/endpoints/los_records.md [hunk 4]
```diff
@@ -12,25 +12,25 @@
...
-created_at           | [Time](/reference/enums#formats) | Read       | LOS record's create time.
+created_at           | [Time](/reference/enums#formats) | Read       | LOS record's creation time.
 {: class="table table-bordered"}
   </div>
 </div>
 
 ## List LOS records
 
-List all LOS records for given account(s).
+List all LOS records for (a) given account(s).
```
Comment (Azdaroth) [inline content/reference/endpoints/los_records.md]: #grammarNazi

**Comment by Fredar** [comment]
@Phazz you have pending changes here

**Comment by Phazz** [comment]
@Azdaroth I addressed your comments.

File: content/reference/endpoints/los_records.md [hunk 5]
```diff
@@ -12,25 +12,25 @@
...
-max_occupancy        | Integer | Read       | Maximum occupancy for which the LOS record is valid.
-original_currency    | String  | Read       | Currency in which the rates are originating from.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
-currency             | String  | Read       | Currency in which the rates are in the response.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
-exchange_rate        | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`.
+day                  | [Date](/reference/enums#formats) | Read       | Day for which this LOS record is valid.
+min_occupancy        | Integer | Read       | Minimum occupancy for which this LOS record is valid.
+max_occupancy        | Integer | Read       | Maximum occupancy for which this LOS record is valid.
+original_currency    | String  | Read       | Original currency that LOS record was generated with (useful when using `exchange_to_currency` query param).<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
```
Comment (Azdaroth) [inline content/reference/endpoints/los_records.md]: I would even add write it that way `useful when using `exchange_to_currency` query param` -> `useful when using `exchange_to_currency` query param as a reference currency`

Comment (Phazz) [inline content/reference/endpoints/los_records.md]: @Azdaroth You can go ahead and make the changes, will be more efficient.

Comment (Phazz) [inline content/reference/endpoints/los_records.md]: @Azdaroth Did the change.

**Comment by Phazz** [comment]
@Azdaroth conflicts resolved

**Pull request merged**